models/klaviyo__flows.sql

```diff
@@ -0,0 +1,37 @@
+with flow as (
```

Comment:
are you going to create a similar model for `klaviyo__campaigns`?

Reply:
yep!